openapi-spec/swagger.yaml

```diff
@@ -1188,6 +1193,131 @@ paths:
           description: The resource does not exist.
         '500':
           description: The fileshare can not be deleted. please check status
+
+  '/v1/{tenantId}/tiers':
+    parameters:
+      - name: tenantId
+        in: path
+        description: The tenant UUID in a multi-tenancy environment.
+        required: true
+        schema:
+          type: string
+    post:
```

Comment:
what is Id in the request body? 
![image](https://user-images.githubusercontent.com/48085413/123680525-72401c00-d866-11eb-9a4f-03924815b2b1.png)

Reply:
Id, if you want to give your own ID.

Reply:
Is it implemented the same way in backend? Because, a new user provided ID should never be used unless it is coming from GET api